lib/developer_portal/lib/liquid/tags/stripe_form.rb

```diff
@@ -5,7 +5,9 @@ class StripeForm < Liquid::Tags::PaymentGatewayBaseForm
 
       desc "Renders the stripe form"
       def render(context)
-        render_erb context, "payment_gateways/stripe", text: @text
+        account = context.registers[:site_account]
```

Comment:
Are you sure that this always contents the `site_account` and it can never be nil? otherwise it would be better something like `account = context.registers[:site_account] || Account.new` to don't break in the next line

Reply:
Yes! https://github.com/3scale/porta/blob/3f7fa1dc5b4904d1e9cbcb7b50436ea01c6c6322/lib/liquid/template_support.rb#L133